### PR TITLE
RSS-ECOMM-2_08: Add a button to navigate to the registration page

### LINKED ISSUE #62: RSS-ECOMM-2_08: Add a button to navigate to the registration page (5 points) 🖱️
Add a button or link on the login page that allows users to navigate to the registration page, making it easy for users to create a new account if they don't have one.

**Acceptance Criteria** ✔️
- [x] A clear and prominent button or link is present on the login page to navigate to the registration page.
- [x] The button or link is functional and directs the user to the registration page.
- [x] Proper handling of navigation and user experience during this process.

### PR DESCRIPTION
**Pull request by Tetiana-KET**
#### 🤔 This is a ...

- [x] 🌟 New task

#### Description

Add a button or link on the login page that allows users to navigate to the registration page, making it easy for users to create a new account if they don't have one.

Acceptance Criteria ✔️

- [x] A clear and prominent button or link is present on the login page to navigate to the registration page.
- [x]  The button or link is functional and directs the user to the registration page.
- [x]  Proper handling of navigation and user experience during this process.

#### Additional Information

- **Screenshots/Links:**
image

#### Checklist

- [x] ✅ I have performed a self-review of my own code.
- [x] 📝 I have commented my code, particularly in hard-to-understand areas.
- [x] 🔧 I have made corresponding changes to the documentation (if applicable).
- [x] 🚫 My changes generate no new warnings or errors.
- [x] 👌 Have at least 1 approve.
- [x] 🌳 Branch from `develop`/`release/...` and target to `develop`/`release/...`.

### CODE REVIEW

File: src/pages/Login/components/LoginForm.tsx [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+import LoginIcon from '@mui/icons-material/Login';
 
 const formStyles = {
   form: { display: 'flex', flexDirection: 'column', width: '100%', height: '100%' },
-  submitButton: { width: 100 },
-  textField: { minWidth: 410 },
+  submitButton: { maxWidth: '160px', align: 'center' },
+  textField: { width: '100%' },
   divider: { height: '1px', backgroundColor: 'black' },
+  buttonContainer: { display: 'flex', justifyContent: 'space-between', marginTop: '1em' },
```
Comment (JsPowWow) [inline src/pages/Login/components/LoginForm.tsx]: buttonContainer  is not used

File: src/pages/Login/components/LoginForm.tsx [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 import { loginFormSchema } from '@core/validation/user-login.validation.schema';
+import LoginIcon from '@mui/icons-material/Login';
 
 const formStyles = {
   form: { display: 'flex', flexDirection: 'column', width: '100%', height: '100%' },
-  submitButton: { width: 100 },
-  textField: { minWidth: 410 },
+  submitButton: { maxWidth: '160px', align: 'center' },
+  textField: { width: '100%' },
   divider: { height: '1px', backgroundColor: 'black' },
```
Comment (JsPowWow) [inline src/pages/Login/components/LoginForm.tsx]: divider is not used

File: src/pages/Login/components/LoginForm.tsx [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
 import { z } from 'zod';
 import { loginFormSchema } from '@core/validation/user-login.validation.schema';
+import LoginIcon from '@mui/icons-material/Login';
 
 const formStyles = {
   form: { display: 'flex', flexDirection: 'column', width: '100%', height: '100%' },
-  submitButton: { width: 100 },
-  textField: { minWidth: 410 },
+  submitButton: { maxWidth: '160px', align: 'center' },
+  textField: { width: '100%' },
```
Comment (JsPowWow) [inline src/pages/Login/components/LoginForm.tsx]: textField is not used

File: src/pages/Login/components/LoginForm.tsx (hunk 1 above)
Comment (Tetiana-KET) [reply src/pages/Login/components/LoginForm.tsx]: Really, I delete the component and forgot about styles, thank you

File: src/pages/Login/components/LoginForm.tsx (hunk 2 above)
Comment (Tetiana-KET) [reply src/pages/Login/components/LoginForm.tsx]: Thank you for your attentiveness

File: src/pages/Login/components/LoginForm.tsx (hunk 3 above)
Comment (Tetiana-KET) [reply src/pages/Login/components/LoginForm.tsx]: Thanks

File: src/pages/Login/components/LoginForm.tsx [hunk 4]
```diff
@@ -1,14 +1,16 @@
-import { Box, Button, Typography } from '@mui/material';
+import { Button, Stack, Typography } from '@mui/material';
 import { FormContainer, TextFieldElement, PasswordElement } from 'react-hook-form-mui';
 import { zodResolver } from '@hookform/resolvers/zod';
 import { z } from 'zod';
 import { loginFormSchema } from '@core/validation/user-login.validation.schema';
+import LoginIcon from '@mui/icons-material/Login';
 
 const formStyles = {
   form: { display: 'flex', flexDirection: 'column', width: '100%', height: '100%' },
```
Comment (JsPowWow) [inline src/pages/Login/components/LoginForm.tsx]: form is not used

Comment (Tetiana-KET) [reply src/pages/Login/components/LoginForm.tsx]: I delete the whole formStyles object - it is not necessary i believe.

**Review by JsPowWow — APPROVED** [review]

**Pull request merged**